service/core.go

```diff
@@ -23,10 +23,10 @@ const indexPage = `
 <html>
   <title>ZQD daemon</title>
   <body style="padding:10px">
-    <h2>ZQD</h2>
-    <p>A <a href="https://github.com/brimdata/zed/tree/main/cmd/zed/lake/serve">zqd</a> daemon is listening on this host/port.</p>
+    <h2>zed lake serve</h2>
+    <p>A <a href="https://github.com/brimdata/zed/tree/main/cmd/zed/lake/serve">zed lake serve</a> process is listening on this host/port.</p>
```

Comment:
Might be a debate here but IMHO the "what" is zed lake service, `zed lake serve` is the command to init a zed lake service.
```suggestion
    <p>A <a href="https://github.com/brimdata/zed/tree/main/cmd/zed/lake/serve">zed lake service</a> is listening on this host/port.</p>
```

Reply:
@mattnibs: No controversy here. I was already on the fence about "serve" vs. "service" and am happy to flip it, so I just pushed the change and will update the screenshot in the PR. Thanks for the approval!